release-notes/release-6.18.md

```diff
@@ -0,0 +1,101 @@
+# Eclipse Che 6.18.0 Release Notes
+
+The Eclipse Che 6.18.0 release contains the following notable features:
+ 
+* Add VueJS language support
+* Improvements for Physical Volume Claim (PVC) handling
+
+---
+
+Eclipse Che 6.18.0 is out now, complete with a new Language Server for VueJS,
+and a number of improvements related to workspace storage and PVC handling.
+
+---
+
+## Quick Start
+
+Che is a cloud IDE and containerized workspace server - get started on:
+
+* Kubernetes ([single-user](https://www.eclipse.org/che/docs/kubernetes-single-user.html) or [multi-user](https://www.eclipse.org/che/docs/kubernetes-multi-user.html))
+* OpenShift ([single-user](https://www.eclipse.org/che/docs/openshift-single-user.html) or [multi-user](https://www.eclipse.org/che/docs/openshift-multi-user.html))
+* Docker ([single-user](https://www.eclipse.org/che/docs/docker-single-user.html) or [multi-user](https://www.eclipse.org/che/docs/docker-multi-user.html))
+* Try Eclipse Che live at [https://che.openshift.io](https://che.openshift.io)
+
+Learn more in our [documentation](https://www.eclipse.org/che/docs/infra-support.html) and start using a shared Che server or local instance today.
+
+---
+
+## Add VueJS language support
+
+Thank you to [@demsking](https://github.com/demsking) for a pull
+request which adds support for [VueJS](https://vuejs.org/) to the
+[Web plugin](https://github.com/eclipse/che/tree/master/plugins/plugin-web).
+Vue.js is a Javascript framework for building user interfaces. This support
+includes syntax highlighting and Intellisense.
+
+**Pull Request:**
+
+* Web Plugin: Add VueJS Language Support: https://github.com/eclipse/che/pull/12470
+---
+
+## Improvements for Physical Volume Claim (PVC) handling
+
+![Ephemeral mode switch](https://user-images.githubusercontent.com/1611939/51479083-f50b4580-1d95-11e9-9eb7-0b0e3351cda8.png)
+
+This release contains a number of changes which improve PVC handling for
+users. We have made it easier for users to use ephemeral storage for
+workspaces by adding an option to enable ephemeral mode in the workspace
+creation page. In addition, we have added the ability to define the PVC
+strategy in the Kubernetes or OpenShift deployment scripts.
+
+If you have ever had a workspace fail to start because of a PVC issue, you
```

Comment:
Well, AFAIU made changes are only about PVCHelper pod logs (it is disabled on many Che Server configurations) but not about PVC issues logs. Actually, it's publishing output of commands like `mkdir /workspace123/projects`, `rm -rf /workspace2432`. And it won't help users investigate a cause of failing if workspace start because of PVC binding issues. So, maybe it would be even better not to list these changes in release notes.
@amisevsk Am I right?

Reply:
Yup, it's mostly for our use.